### PR TITLE
Commentary for Factories

### PR DESCRIPTION
**Pull request by adesola15**

### CODE REVIEW

File: tests/domains/user/test_user.py [hunk 1]
```diff
@@ -19,274 +16,49 @@ def test_create_user(self, client, admin_user) -> None:
...
-
-        assert response.status_code == 201
-        assert data.get("message") == f"User with email {user.email} successfully created"
+        # ... (same structure as above)
 
     def test_create_user_with_missing_argument(self, client, admin_user) -> None:
+        # Test creating a user with missing arguments
         user = UserFactory.build()
-
-        params = {
```
Comment (sebdeol) [inline tests/domains/user/test_user.py:50]: You've removed the test, so this function now does nothing

Comment (sebdeol) [reply tests/domains/user/test_user.py:50]: I'll add it back and the ones below

Comment (adesola15) [reply tests/domains/user/test_user.py:50]: Ah sorry I made a mistake when copying and pasting the commentary

Comment (sebdeol) [reply tests/domains/user/test_user.py:50]: No problem, just fixed it! It should all be green and ready to merge now :)

**Review by sebdeol — APPROVED** [review]

**Pull request merged**